interface/src/app/sample/services.spec.js

```diff
@@ -0,0 +1,63 @@
+describe('adage.sample.services', function() {
+  var $httpBackend, $log, Sample, mockSampleData;
+
+  beforeEach(module('adage.sample.services'));
+  beforeEach(module('adage.mocks.sample'));
```

Comment:
Nice. It was handy to have these on separate lines as I refactored and tested the tests, but I like the combined call for the final product.

- [ ] combine `module` calls as @dongbohu suggests